react/react1/week2/todo-list/src/Ul.jsx

```diff
@@ -0,0 +1,31 @@
+import React,{ useState}  from 'react';
+
+import TodoList from "./List";
+import todoList  from './data';
+<todoList todoList = {todoList}/>
+
+export default function Ul () { 
+  const  [list,setList] = useState(todoList);
+
+  const addTaskClick= ()=> {
+    const arryLen = list.length;
+    const newTodo = `Todo number ${arryLen + 1} has been Added`;
+    const newTodoList = list.concat([{ id: '' + (arryLen + 1) , description: `${newTodo}`}]);
```

Comment:
an elegant way of adding a new item is destructuring the array state and simply typing the new content, fx:

```
setState([...state, {
id: 1,
task: "do the laundry"
}])
```